supported_sites.json

```diff
@@ -1,5 +1,20 @@
 [
 {
+  "url":"http://thenoobcomic.com/",
+  "title":"The Noob",
+  "imageUrl":"http://thenoobcomic.com/wp-content/uploads/2015/07/00001.jpg",
+  "imageSelector":"#comic img",
+  "imageIndex":"0",
+  "firstSelector":".comic-nav-first",
+  "firstIndex":"0",
+  "prevSelector":".comic-nav-previous",
+  "prevIndex":"0",
+  "nextSelector":".comic-nav-next",
```

Comment:
This selector doesn't seem to exist either. I think you can replace it with .navi-next.